features/step_definitions/migration/migration_steps.rb

```diff
@@ -0,0 +1,128 @@
+Given(/^At least (\d+) Xen Hypervisors are available$/) do |arg1|
+  # get hypervisors and count Xen's ones - for migration at least 2
+  hashypervisor_xen = 0
+  @hypervisors = control_node.openstack.hypervisor.list
+  expect(@hypervisors).not_to be_empty
+
+  @hypervisors.each do |h|
+    hypervisor = control_node.openstack.hypervisor.show(h.id)
+    hashypervisor_xen = (hashypervisor_xen + 1) if hypervisor.hypervisor_type == "Xen"
+  end
+  expect(hashypervisor_xen).to be >= arg1.to_i
+end
+
+Given(/^At least (\d+) KVM Hypervisors are available$/) do |arg1|
+  # get hypervisors and count KVM's ones - for migration at least 2
+  hashypervisor_kvm = 0
+  @hypervisors = control_node.openstack.hypervisor.list
+  expect(@hypervisors).not_to be_empty
+
+  @hypervisors.each do |h|
+    hypervisor = control_node.openstack.hypervisor.show(h.id)
+    hashypervisor_kvm = (hashypervisor_kvm + 1) if hypervisor.hypervisor_type == "QEMU"
+  end
+  expect(hashypervisor_kvm).to be >= arg1.to_i
+end
+
+When(/^I create an KVM instance$/) do
+  def delete_old_instances(server_id)
+    puts("old instance #{server_id } found - deleting")
+    control_node.exec!("openstack server delete #{server_id}")
+  end
+  ## TODO add --key-name zkubala
+  # clean old instances
+  @servers = control_node.openstack.server.list
+  @instance_name = "kvm_mig_instance"
+  @servers.each do |s|
+    s.name == @instance_name and delete_old_instances(s.id)
+  end
+  # create a new instance
+  new_instance = control_node.exec!("openstack server create -f shell --flavor m1.smaller --image jeos --security-group default -c id #{@instance_name}")
+  @new_instance_id = new_instance.output[4, 36]
```

Comment:
Is this needed? The output only should contain the server id (looking at -c id from the command above).